charts/loki-distributed/values.yaml

```diff
@@ -306,6 +306,8 @@ ingester:
               matchLabels:
                 {{- include "loki.ingesterSelectorLabels" . | nindent 12 }}
             topologyKey: failure-domain.beta.kubernetes.io/zone
+  # -- Pod Disruption Budget maxUnavailable
+  maxUnavailable: 0
```

Comment:
That's the intention @trevorwhitney. We would like to disable PDB because Karpenter respects the PDB and does not evict those pods with PDB enabled. I think would be a great idea if the user could decide whether he wants to have PDB enabled or not.